src/components/atoms/Button/style.scss

```diff
@@ -15,12 +15,12 @@
         outline: none;
     }
 
-    &-link {
+    &--link {
         right: 25px;
         top: 338px;
     }
 
-    &-sublink {
+    &--sublink {
```

Comment:
link and sublink is actually is a element, so better use `__element`

Reply:
Ok, changed